Build: "make run" fails when launching the web server under native Windows PowerShell

### What happened?

I attempted to set-up the development environment under Windows 11 with PowerShell 5.1 and 7.6.4. The results are similar in both. Compilation, migrations run successfully but we see an error when launching the web server, which results in it not being available:

```
PS C:\dev\grafana> make run
"C:\Users\harol\AppData\Local\go-build\a0\a09161d6f7d0c00c13254525f612276483cedc9a571bee997f4b45db6cc6e470-d\air.exe" -c .air.toml

  __    _   ___
 / /\  | | | |_)
/_/--\ |_| |_| \_ v1.62.0, built with Go go1.26.5

[17:31:47] watching apps
[17:31:47] watching apps\advisor
[17:31:47] watching apps\advisor\kinds
[17:31:47] watching apps\advisor\kinds\cue.mod
[17:31:47] watching apps\advisor\pkg

(.....)
[17:33:12] running...
[17:33:12] CMD will not recognize non .exe file for execution, path: C:\dev\grafana\bin\grafana-air server -profile -profile-addr=127.0.0.1 -profile-port=6000 -profile-block-rate=1 -profile-mutex-rate=5 -packaging=dev cfg:app_mode=development
diagnostics: pprof profiling enabled addr 127 port 6060 blockProfileRate 1 mutexProfileRate 0
INFO [08-08|17:33:14] Starting Grafana                         logger=settings version=13.2.0-local commit=bad5c890fb92a1b68a267c741d7eda692723164f branch=main compiled=2026-01-01T00:00:00+01:00

(.....)

INFO [08-08|17:33:15] Loading plugins...                       logger=plugin.store
panic: listen tcp: lookup 127: no such host

goroutine 227 [running]:
github.com/grafana/grafana/pkg/server/bootstrap.setupProfiling.func1()
        C:/dev/grafana/pkg/server/bootstrap/diagnostics.go:151 +0x198
created by github.com/grafana/grafana/pkg/server/bootstrap.setupProfiling in goroutine 1
        C:/dev/grafana/pkg/server/bootstrap/diagnostics.go:144 +0x42d
[17:33:17] Process Exit with Code: 1
```

**Root cause**
When passing the launch parameters the IP address is not escaped nor wrapped, which causes the 127.0.0.1 to be separated into segments (and apparently dropping the rest of parameters also), as exposed by this line:
`diagnostics: pprof profiling enabled addr 127 port 6060 blockProfileRate 1 mutexProfileRate 0`
So as pprof can not bind, the launch fails and the server ends up unreachable.


### What did you expect to happen?

The server to execute and bind to 127.0.0.1:3000 without errors when launched via PowerShell.

### Did this work before?

No, as far as I could see.

### How do we reproduce it?

1. Download the grafana repository in a Windows 11 machine, natively (not under WSL)
2. Set up the prerrequisites
3. Execute make run


### Is the bug inside a dashboard panel?

_No response_

### Environment (with versions)?

Grafana: Current main branch
OS: Windows 11 (10.0.26200)
Browser: N/A - server does not start
PowerShell: 5.1.26100.8875 & 7.6.4


### Grafana platform?

None

### Datasource(s)?

_No response_

--- a/.air.toml
+++ b/.air.toml
@@ -1,6 +1,6 @@
 [build]
 bin = "./bin/grafana-air"
-args_bin = ["server", "-profile", "-profile-addr=127.0.0.1", "-profile-port=6000", "-profile-block-rate=1", "-profile-mutex-rate=5", "-packaging=dev", "cfg:app_mode=development"]
+args_bin = ["server", "-profile", "-profile-addr='127.0.0.1'", "-profile-port=6000", "-profile-block-rate=1", "-profile-mutex-rate=5", "-packaging=dev", "cfg:app_mode=development"]
 cmd = "make CGO_ENABLED=0 SOURCE_DATE_EPOCH=1767222000 GO_BUILD_DEV=1 build-air"
 exclude_regex = ["_test.go", "_gen.go"]
 exclude_unchanged = true

